src/index.ts

```diff
@@ -11,12 +11,12 @@ export {
 } from "./cover_crypt/key_generation"
 export { type CoverCryptHybridDecryption } from "./cover_crypt/decryption"
 export { type CoverCryptHybridEncryption } from "./cover_crypt/encryption"
+export { type Policy, PolicyAxis } from "./cover_crypt/interfaces/policy"
```

Comment:
```suggestion
export { type Policy, type PolicyAxis } from "./cover_crypt/interfaces/policy"
```

Reply:
What does it change ?

Reply:
Because you don't want the user to do `new PolicyAxis()`, you want to have a `let { PolicyAxis } = await CoverCrypt()` before the construction to load the wasm.